docs/src/components/mailing-list.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
Since this component doesn't use any hooks, should we move this directive down to the `MailingListForm`?

Reply:
I'd recommend leaving the `use client` here so that e.g. if we have this mailing list component on many pages, when users do a client-side navigation, all the markup in this component doesn't have to be downloaded again

Reply:
Ahh nice, that makes sense, thanks!